themes/learn.jquery.com/content-page.php

```diff
@@ -20,7 +20,7 @@
 		<div class="bottom-links row">
 			<?php if (isset($next_prev_arr['prev'])): ?>
 				<div class="prev six columns">
-					<a href="<?php echo $next_prev_arr['prev']->guid; ?>">
+					<a href="<?php echo get_page_link( $next_prev_arr['prev']->ID ); ?>">
```

Comment:
Yep I'll remove the changes. I was actually trying to test @sigorilla's changes forgetting that my commit will be attached to this PR